.golangci.yml

```diff
@@ -32,7 +32,7 @@ linters:
     - errcheck
     - gocritic
     - gofmt
-    - goimports
+    - gci
```

Comment:
This is incorporated into the `.github` sync workflow in https://github.com/cloudquery/.github/pull/107